specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp

```diff
@@ -193,6 +204,14 @@ model ChatCompletions {
   @projectedName("json", "choices")
   choices: ChatChoice[];
 
+  @doc("""
+  Content filtering results for zero or more prompts in the request. In a streaming request, 
+  results for different prompts may arrive at different times or in different orders.
+  """)
+  @added(ServiceApiVersions.v2023_06_01_Preview)
+  @projectedName("json", "promptFilterResults") // TODO: or "prompt_filter_results"?
```

Comment:
What determines the `projectedName`? Some places use camel case but others were snake case.

Reply:
The first string value of the decorator tells you where the second string value will be used as a name for this field. So basically, what determines the second value is the convention of a particular language. For `"json"` I think it should be snake casing, for C# pascal casing and Java camel casing.
If you don't define a projected name for a specific language then the value of the field in TSP will be used as the default.

Reply:
Right--in a future change, we'll merge the JSON projected name (wire format) with the field name, but for the time being we're snapping to the naming conventions suggested by the compiler to limit warning spew or a matching repetitive need for warning suppression.

So in this case, since the field shows up as `prompt_filter_results` in the incoming JSON, you'd want `projectedName("json", "prompt_filter_results") promptFilterResults ...`